src/components/Menu/SubMenu.tsx

```diff
@@ -107,35 +107,20 @@ export const SubMenu = ({
                 target={el.link.external ? '_blank' : '_self'}
                 onClick={(event) => {
                   event.stopPropagation();
-                  el.onClick();
+                  typeof el.onClick === 'function' && el.onClick();
```

Comment:
Yup, basically the example above was the Link -> This would be a regular MenuItem (which does not redirect to a new page). The `typeof el.onClick === 'function' &&` was added to make sure, this triggers only when onClick is being provided.